<launch>
    <node name="control_to_servo"  pkg="me134_final" type="control_to_servo.py" output="screen" >
    </node>

    <!--node name="joystick_controller"  pkg="me134_final" type="joystick_controller.py" output="screen" >
    </node-->

    <node name="servo_hat_commander"  pkg="me134_final" type="servo_hat_commander.py" output="screen" >
        <param name="servo_min_pwm" value="50" />
        <param name="servo_max_pwm" value="3950" />
    </node>


</launch>
